Tests/SecureXPCTests/XPCEncoder/Array encoding tests.swift

```diff
@@ -0,0 +1,232 @@
+import XCTest
+@testable import SecureXPC
+
+final class XPCEncoder_ArrayEncodingTests: XCTestCase {
+	// MARK: Signed Integers
+	
+	func testEncodes_arrayOf_SignedIntegers_asArrayOf_XPCInts() throws {
+		let   ints: [Int  ] = [.min, -123, 0, 123, .max]
+		let  int8s: [Int8 ] = [.min, -123, 0, 123, .max]
+		let int16s: [Int16] = [.min, -123, 0, 123, .max]
+		let int32s: [Int32] = [.min, -123, 0, 123, .max]
+		let int64s: [Int64] = [.min, -123, 0, 123, .max]
+		
+		try assert(  ints, encodesEqualTo: createXPCArray(from:   ints, using: { xpc_int64_create(Int64($0)) }))
+		try assert( int8s, encodesEqualTo: createXPCArray(from:  int8s, using: { xpc_int64_create(Int64($0)) }))
+		try assert(int16s, encodesEqualTo: createXPCArray(from: int16s, using: { xpc_int64_create(Int64($0)) }))
+		try assert(int32s, encodesEqualTo: createXPCArray(from: int32s, using: { xpc_int64_create(Int64($0)) }))
+		try assert(int64s, encodesEqualTo: createXPCArray(from: int64s, using: { xpc_int64_create(Int64($0)) }))
```

Comment:
This is code-genned code. I didn't add a special case for `Int64`.

At worst case, this will be a no-op integer conversion. I'd be surprised if the compiler couldn't optimize it, and even if it doesn't, it wouldn't really make much of an impact on performance anyway.

However, I think it's kind of nice to keep it. I wrote these test cases in a style that tries to optimize for visual conformity, so that I can leverage the human proclivity to visually detect outliers, to detect unintentional/bad differences